src/Table/Table.stories.jsx

```diff
@@ -77,4 +88,24 @@ storiesOf('Table', module)
       caption="Famous Internet Cats"
       className={['table-responsive']}
     />
-  ));
+  ))
+  .add('sortable', () => {
+    const catDataSortable = catData.slice();
+
+    return (<Table
+      data={catDataSortable.sort((firstElement, secondElement) => sort(firstElement, secondElement, catColumns[0].key, 'desc'))}
```

Comment:
This just performs an initial sort so that the table is initially displayed in some kind of default sorted order. Something we should consider is whether or not we expect the user to pass in pre-sorted data. I assume yes, since we don't want to do client-side sorting, but I just wanted to double-check.